src/content/development/release-process/_index.en.md

```diff
@@ -65,16 +65,16 @@ for the job listed but the important one is `Release Images`. When complete, the
 green check mark on success or a red X on failure. A failure likely means the artifacts were not published to Quay, in
 which case select the failed check, inspect the logs, correct the issue and re-run the job.
 
-### Step 0: Create a stable branch (if necessary)
+### Sometimes: Create stable branches for stable releases
 
-If you're creating a new stable release, the release automation process can create the stable branches. To do so,
-navigate to the [releases](https://github.com/submariner-io/releases) repository and fork it.
+If you're creating release that will receive backports, a stable release not a milestone release, you need to create stable branches for
+each repository. The release automation process can do this for you. To do so, navigate to the
+[releases](https://github.com/submariner-io/releases) repository.
 
-1) Create a new branch for the intended release.
+1) Create a new file in the `releases` directory (you can copy the `example.yaml` file). For our example, we'll name it `v0.8.0.yaml`.
 
-2) Create a new file in the `releases` directory (you can copy the `example.yaml` file). For our example, we'll name it `v0.8.0.yaml`.
-
-3) Fill in the general fields for the release with the `status` field set to `branch`.
+2) Fill in the `version`/`name` fields for the release. The `branch` field is optional for releases targeting the default `devel` branch.
+￼  The `status` field must set to `branch` for this phase.
```

Comment:
If we are creating a stable branch the `branch`field will never be optional, that's confusing ':D

Reply:
Ah good catch, that was an accidental leftover from a previous attempt (#533).

Reply:
Fixed :+1: